### PR TITLE
docs(cli): Remove outdated example

### LINKED ISSUE #8595: CLI keptn configure monitoring example text outdated
**Environment**

* Client OS (e.g., Linux, macOS, Windows):
* Keptn Version (`keptn version`):  0.18.1
* Kubernetes Cloud Provider (e.g., GKE, AKS):
* Kubernetes version (`kubectl version`):

**Affected Component**

* [ ] Approval
* [ ] Bridge
* [x] CLI
* [ ] Datastore
* [ ] Docs
* [ ] Distributor / Event Broker
* [ ] Helm
* [ ] Istio
* [ ] Jmeter
* [ ] Kubernetes Integration
* [ ] Openshift Integration
* [ ] REST API
* [ ] Webhook

**Describe the bug**
`keptn configure monitoring --help` shows some examples which include one:

```
Examples:
keptn configure monitoring dynatrace --project=PROJECTNAME
```

This is invalid as the command requires a service.

**To fix**
Remove this example line from the CLI examples command. The examples **with** a `--service` flag should remain as they are still valid.

### PR DESCRIPTION
**Pull request by lunarwhite**
## This PR

Remove the outdated `keptn configure monitoring` example text.

### Related Issues

Fixes #8595

### CODE REVIEW

File: cli/cmd/configure_monitoring.go [hunk 1]
```diff
@@ -40,8 +40,7 @@ var monitoringCmd = &cobra.Command{
 **Note:** If you are executing *keptn configure monitoring dynatrace*, the service flag is optional since Keptn automatically detects the services of a project. 
 See https://keptn.sh/docs/install/monitoring/ for more information.
 `,
-	Example: `keptn configure monitoring dynatrace --project=PROJECTNAME
-keptn configure monitoring prometheus --project=PROJECTNAME --service=SERVICENAME
+	Example: `keptn configure monitoring prometheus --project=PROJECTNAME --service=SERVICENAME
```
Comment (thisthat) [inline cli/cmd/configure_monitoring.go:43]:
```suggestion
	Example: `keptn configure monitoring dynatrace --project=PROJECTNAME --service=SERVICENAME`
`keptn configure monitoring prometheus --project=PROJECTNAME --service=SERVICENAME
```

**Review by thisthat — CHANGES_REQUESTED** [review]

File: cli/cmd/configure_monitoring.go [hunk 2]
```diff
@@ -40,8 +40,7 @@ var monitoringCmd = &cobra.Command{
 **Note:** If you are executing *keptn configure monitoring dynatrace*, the service flag is optional since Keptn automatically detects the services of a project. 
 See https://keptn.sh/docs/install/monitoring/ for more information.
 `,
-	Example: `keptn configure monitoring dynatrace --project=PROJECTNAME
-keptn configure monitoring prometheus --project=PROJECTNAME --service=SERVICENAME
```
Comment (thisthat) [inline cli/cmd/configure_monitoring.go:44]: you are now missing the example for prometheus. Please, re-add this line

**Review by thisthat — CHANGES_REQUESTED** [review]

File: cli/cmd/configure_monitoring.go (hunk 2 above)
Comment (lunarwhite) [reply cli/cmd/configure_monitoring.go:44]: Ok, I'll fix it.

**Review by thisthat — APPROVED** [review]
LGTM! Thank you @lunarwhite for your contribution!

**Pull request merged**